scripts/projectView.js

```diff
@@ -10,29 +12,28 @@ projectView.handleMainNav = function() {
   $('.main-nav .tab:first').click();
 };
 
+//HIDES ALL BUT THE BEGINNING OF EACH PROJECT, SHOWS ALL WHEN CLICKED AGAIN
 projectView.setTeasers = function() {
   $('.project-picture').hide();
-
   $('#projects').on('click', '.see-more', function(e) {
```

Comment:
I wonder if you could combine these two event handlers into one, using an if statement. Maybe, if class = .see-more, run the code in the first event handler, and if class = .see-less, run the code in the second event handler.